config.sh

```diff
@@ -8,26 +8,28 @@ VERSION="1.0"
 BUILD_DIR=".build-linux"
 BRIDGE_APP_NAME="containerbridge"
 DATABASE_NAME="TodoListCloudantDatabase"
-REGISTRY_URL="registry.ng.bluemix.net"
+REGISTRY_URL="registry.eu-gb.bluemix.net"
 DATABASE_TYPE="cloudantNoSQLDB"
 DATABASE_LEVEL="Lite"
+NAME_SPACE="todolist_space"
 
 function help {
   cat <<-!!EOF
 	  Usage: $CMD [ build | run | push-docker ] [arguments...]
 
 	  Where:
 	    install-tools				Installs necessary tools for config, like Cloud Foundry CLI
-	    login					Logs into IBM Cloud and Container APIs
+      config-cli                      Sets up and configures necessary CLI tools
+	    login					Logs into Bluemix and Container APIs
+      setup <clusterName>                           Sets up the clusters
 	    build <imageName>          			Builds Docker container from Dockerfile
-	    run   <imageName>         			Runs Docker container, ensuring it was built properly
+        run   <imageName>                     Runs Docker container, ensuring it was built properly
 	    stop  <imageName> 				Stops Docker container, if running
 	    push-docker <imageName>			Tags and pushes Docker container to IBM Cloud
-	    create-bridge				Creates empty bridge application
-	    create-db				        Creates database service and binds to bridge
-	    deploy <imageName>				Binds everything together (app, db, container) through container group
+	    create-db <imageName>				        Creates database service
+	    deploy				Binds everything together (app, db, container) through container group
 	    populate-db	<imageName>			Populates database with initial data
-	    delete <imageName>				Delete the group container and deletes created service if possible
+	    delete <imageName>				Delete the created service and cluster if possible
 	    all <imageName>                 		Combines all necessary commands to deploy an app to IBM Cloud in a Docker container.
```

Comment:
The help commands need to be updated - `all` prompts me for multiple args, and I do not know what the second one is.